rollup.config.js

```diff
@@ -0,0 +1,27 @@
+import rollupConfig from '@ovh-ux/component-rollup-config';
+import image from 'rollup-plugin-img';
+
+const config = rollupConfig({
+  input: 'src/index.js',
+});
+
+const esConfig = config.es();
+esConfig.plugins.push(image());
```

Comment:
Shouldn't be handled directly through shared configuration `components-rollup-config` itself?

Reply:
Make sense.

Reply:
Moved in https://github.com/ovh-ux/component-rollup-config/pull/56

Reply:
`ovh-ux/component-rollup-config` bumped to `5.0.0-beta.9`